tests/test_models.py

```
@@ -161,10 +165,89 @@ def test_embedding_validity():
     with pytest.raises(Exception):
         _ = Embedding(emb_type="temporal", mode="learn", features=128)
 
+    with pytest.raises(Exception):
```

Ruff

`pytest.raises(Exception)` should be considered evil

(B017)